Dockerfile.openssl

```diff
@@ -70,15 +70,12 @@ RUN git clone -b $QAT_ENGINE_VERSION https://github.com/intel/QAT_Engine && \
     make && \
     make install
 
-WORKDIR /
```

Comment:
@mythi 
> what is the reason for this commit? WORKDIR /envoy?

The logic here is that copying everything recursively under “/” might accidentally clash with the distribution’s directories/files.

It doesn’t harm and makes it easier to understand what comes from where.

Also from the point of view of keeping the build output under /build, since bazel doesn’t support when the build output is under the workspace.

Would it make sense?

Reply:
Makes sense but how about making it `/envoy-build` to be explicit that it's separate from envoy submodule?

Reply:
@mythi 
> Makes sense but how about making it /envoy-build to be explicit that it's separate from envoy submodule?

Would be fine, except in connection with storing the build output under /build might be slightly confusing.

Maybe envoy-qat?

Reply:
+1

Reply:
@mythi 
OK, done.